framework/src/modules/chain/chain.js

```diff
@@ -235,7 +235,7 @@ module.exports = class Chain {
 			if (!this.options.loading.rebuildUpToRound) {
 				this.channel.subscribe(
 					'network:event',
-					async ({ data: { event, data, peerId } }) => {
+					async ({ data: { event, data } }) => {
```

Comment:
Does this change mean that `peerId` is at the same level as the data sent by the peer? Just asking.

Reply:
I think original code here was correct.
it `event` and `data` and `peerId` is at the same level

Reply:
Right guys. I got confused because we have two datas in the same line. I've changed the variable name to avoid variable shadowing in the future.